subsys/net/ip/net_pkt.c

```diff
@@ -41,6 +41,13 @@ LOG_MODULE_REGISTER(net_pkt, CONFIG_NET_PKT_LOG_LEVEL);
 #include "net_private.h"
 #include "tcp_internal.h"
 
+/* Make sure net_buf data size is large enough that IPv6
+ * and possible extensions fit to the network buffer
+ */
+#if defined(CONFIG_NET_BUF_FIXED_DATA_SIZE) && defined(CONFIG_NET_NATIVE_IPV6)
+BUILD_ASSERT(CONFIG_NET_BUF_DATA_SIZE >= 96);
```

Comment:
Perhaps describe how `96` is calculated in the comment?

Reply:
Updated the comment